bokeh/__init__.py

```diff
@@ -10,10 +14,9 @@
     __version__ = get_versions()['version']
     del get_versions
 
-
 _notebook_loaded = None
 
-def load_notebook(resources=None, verbose=False, force=False, skip=False):
+def load_notebook(resources=None, verbose=False, skip=False):
```

Comment:
but skip is used here: https://github.com/bokeh/bokeh/blob/master/bokeh/__init__.py#L82
or you mean it is not frequently used??